docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,122 @@
+How to create a package slice for Chisel
+****************************************
+
+You want to build a chiselled ROCK, but Chisel cannot find the exact bits and pieces of the packages you need to install! This means that said package doesn't yet have the slice definitions you're looking for. In these cases, your best move is to actually **create your own slice definition** (which you can, later on, propose to be merged upstream for everyone else to use).
+
+**Let's assume you are trying to create a slice definition for installing the OpenSSL binary into your ROCK!**
+
+Make sure the slice definition doesn't exist yet
```

Comment:
we should not use contractions https://docs.ubuntu.com/styleguide/en

Reply:
well, doesn't is allowed....

Reply:
Can i resolve this conversation?